Benchmark/Tokenizer.cs

```diff
@@ -0,0 +1,32 @@
+﻿using BenchmarkDotNet.Attributes;
+
+using System.Runtime.CompilerServices;
+
+namespace Benchmark;
+
+[SimpleJob]
+public class Tokenizer {
+    static string? text;
+
+    [GlobalSetup]
+    public void Setup() {
+        var filePath = Path.Combine(GetMyPath(), "../..", "README.md");
+        text = File.ReadAllText(filePath);
+        static string GetMyPath([CallerFilePath] string filePath = "") => filePath; // Get the path of this file
+    }
```

Comment:
```suggestion
    static readonly string text = File.ReadAllText(Path.Combine(GetMyPath(), "../..", "README.md"));
    static string GetMyPath([CallerFilePath] string filePath = "") => filePath;
```

Although would be better as an oneliner without the magic compiler attribute if possible.
`static string readmeText => ...;`

Reply:
Good idea. Unfortunately the "magic compiler attribute" is the only reliable way of getting the path.